domains/jonathan.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "JPathmakumar",
+           "email": "",
+           "discord": "1259510650657443941",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.jRCoAGkvoH9rcNGoZEhFoMOpz5yQfPz1ETB6LuUAvQmWCD3FP9kESaZsOrAdQJ5j36X7lz9x-6rgy2kNpT55u9CB0-RRYngtg4pDgkDs1DNAJwY1ai2PhAp88hggHj5byYTpSFE3GbASt1C0am-8-aG3tyCSNsJA8-xpoe5oobz337KB7qmxqzIOl1AtAsLxesepYGdk06pW_7J1tauZT131QLoXi5PPhmqbpHewqy5On-tAhS5T3_6kgsggkooxaHUKx-lz0FqElUc9ZJALlasc6mF2jJM9ppNmae9ISpwX6t92U-1qMnA0qqRwEUCqpyP5g__h9dLGT2OMsrwFkg.5dubPcoUJO_2yle1z1N4aQ.5k2pg4PtOI6aPevtlnZn8NKwnoEqVcfyZGe_6-OL4bAI3dyKCihA78nUrXG6wayR9jpd_GhrbwHl037kTCYpeDUNDo-MaVLcsNT6c7L3b5yFH4K_oF_KeN6ooglNCqpw.ogx1ApDK1zcQI0CYTY4U_g"
+        },
+    
+        "record": {
+            "CNAME": "https://github.com/jpathmakumar/portfolio-website"
```

Comment:
Invalid CNAME record, CNAME records must be hostnames with nothing else attached. See docs: https://docs.is-a.dev